crates/storage-proto/proto/dev/restate/storage/v1/domain.proto

```diff
@@ -336,8 +336,9 @@ message OutboxMessage {
 
     message OutboxIngressResponse {
         FullInvocationId full_invocation_id = 1;
-        SocketAddr socket_addr = 2;
+        /*SocketAddr socket_addr = 2;*/
```

Comment:
Should we mark `2` as reserved (https://protobuf.dev/programming-guides/proto3/#fieldreserved)?

Reply:
I'll do that, but we are backward incompatible anyway at the moment :(

Reply:
True. Then I'd also be fine with reusing `2` (given that we require a wipe anyway).